share/examples/bhyve/vmrun.sh

```diff
@@ -139,7 +139,7 @@ vncsize=${DEFAULT_VNCSIZE}
 tablet=""
 
 # arm64 only
-uboot_firmware="/usr/local/share/u-boot/u-boot-bhyve-arm64/u-boot.bin"
```

Comment:
We have u-boot-bhyve-arm64 for the native ABI, and that's what bsdinstall will install

Reply:
Oh, ok. I assume there's no practical difference between the hybrid and native packages? I can't see one so far.

I'll drop this commit, thanks.

Reply:
Indeed, both cross-compile for plain arm64

Reply:
Still doesn't show for me as installable FWIW:

```
> pkg64c search bhyve
bhyve-rc-4                     FreeBSD RC script for starting bhyve guests in tmux
vm-bhyve-1.5.0                 Management system for bhyve virtual machines
vm-bhyve-devel-1.5.0.17        Management system for bhyve virtual machines
```

Reply:
https://pkg.cheribsd.org/CheriBSD:20240315:aarch64c.html and https://pkg.cheribsd.org/CheriBSD:20230804:aarch64c.html both list it...

Reply:
Only thing I can think of is if you're using the debug package set instead, which doesn't seem to have it for either version?

Reply:
Ah, I'm using debug only for purecap.